utils/src/main/kotlin/com/exactpro/th2/lwdataprovider/MessageSearcher.kt

```diff
@@ -38,38 +36,38 @@ import java.util.function.Function
 class MessageSearcher private constructor(
     private val service: DataProviderService,
     private val searchStep: Duration = DEFAULT_SEARCH_STEP,
-    private val defaultBook: String,
-    private val defaultSessionGroup: String,
-    private val defaultMessageStreams: Set<MessageStream>,
 ) {
+    init {
+        require(!searchStep.isZero) {
+            "'searchStep' can't be zero"
+        }
+        require(!searchStep.isNegative) {
+            "'searchStep' can't be negative"
+        }
+    }
+
     /**
      * Searches the first message in [BASE64_FORMAT] format matched by filter. Execute search from [Instant.now] to previous time
-     * @param sessionGroup is parameter for request. Searcher uses [messageStreams] value when [sessionGroup] is null or blank
+     * @param sessionGroup is parameter for request. Searcher uses [sessionAlias] value when [sessionGroup] is null
      * @param searchInterval for searching. Searcher stops to send requests when the next `from` time is before than search start time minus [searchInterval]
      */
-    @JvmOverloads
     fun findLastOrNull(
```

Comment:
I think we could change the API a bit to make it more user-friendly. Right now the end user will use it like this:
```java
MessageSearcher searcher = /**/
// alias is the same as group
var res = searcher.findLastOrNull("book", "alias", "alias", Direction.FIRST, Duration.ofDays(1));
// alias is not the same as group
var res = searcher.findLastOrNull("book", "group", "alias", Direction.FIRST, Duration.ofDays(1));
```
I  think we might at least provide an overload that accepts only an alias (without session group)